src/spec_tools/json_schema.cljc

```diff
@@ -178,12 +178,13 @@
 (defmethod accept-spec ::visitor/set [dispatch spec children _]
   {:enum children})
 
-(defn- maybe-with-title [schema spec]
-  (if-let [title (st/spec-name spec)]
-    (assoc schema :title (impl/qualified-name title))
-    schema))
+(defn- maybe-with-title [schema spec options]
+  (letfn [(infer-titles? [options] (or (nil? (:infer-titles options)) (:infer-titles options)))]
```

Comment:
I think the logic should be `(-> options :infer-titles false? not)`

Reply:
As stated in the json-schema webpage, these annotations are not required, but recommended as good practices to keep the schemas "self-documented". I would vote to keep it as a default option, and as far as the documentation it easy to acess and provide guidance to disable it, its fine.

I think a docstring would be good in the `transform` fn.

Your suggestion is more concise =) I can change that and add a docstring to `transform`.